fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -389,7 +393,7 @@ protected void resetTranslationChain() {
         if (translationChain == null) {
             translationChain = getTranslationChain();
             final List<Converter<String, String>> newChain =
-                    new ArrayList<>(singleton(new TransactionIdentifierConverter(session)));
+                    new ArrayList<>(singleton(new TransactionIdentifierConverter(transactionService, session)));
```

Comment:
Oh, this is weirder than that. Never mind. Do whatever here and I will send an orthogonal PR at some future point.